database/schema.sql

```diff
@@ -137,7 +137,8 @@ CREATE TABLE loaders (
 	name          character varying(2048),
 	env           json,
 	tags          json,
-    	lastused      timestamp with time zone NOT NULL
+	lastused      timestamp with time zone NOT NULL,
```

Comment:
There is rebase weirdness here because I commented on that line 3 commits ago...